### DISCUSSION TITLE
Questions about custom datasets.

### DISCUSSION THREAD

**Discussion by kx-Z** [open-edge-platform/anomalib#525] [Q&A]
I wonder if these algorithms are effective for some large exceptions? The background may also be slightly different, such as damage on different containers. If it is effective, what parameters need to be adjusted?

**Comment by djdameln** [comment]
Can you provide us with a little bit more information about your use case? For example, how many normal and anomalous images are present in the dataset? Are the images taken from a fixed camera position? Did you try any Anomalib models already and what were the results? It would definitely help if you could share some example images of your dataset.

In general, most anomalib models should be able to handle some variation in the background, but it all depends on the dataset. The training set must capture the natural variation in the normal class, otherwise the model will be likely to generate false positive detections.
